### ISSUE TITLE
Documentation should explain what CommonMark extensions are used

### ISSUE THREAD

**Issue by e00E** [Keats/gutenberg#2030]
# Documentation issue

The Zola website states

> Content is written in [CommonMark](https://commonmark.org/), a strongly defined, highly compatible specification of [Markdown](https://www.markdownguide.org/).

This is not completely correct. For example, CommonMark does not define tables, yet Zola supports them. The changes and additions to CommonMark should be documented.

**Comment by Keats** [comment]
Yes, we should mention we include all the options from https://github.com/raphlinus/pulldown-cmark#pulldown-cmark
